kolibri/plugins/device_management/assets/src/state/actions/contentTransferActions.js

```diff
@@ -34,6 +36,13 @@ export function downloadChannelMetadata(store) {
     .then(completedTask => {
       const { taskId, cancelled } = completedTask;
       if (taskId && !cancelled) {
```

Comment:
This might not be the right place for this logic. 

On https://github.com/learningequality/kolibri/blob/e37e8237ac19bce6e8d214eda5f333057f93a107/kolibri/plugins/device_management/assets/src/views/select-content-page/index.vue#L173

I added a line to update the stale Topic Contents list after an update. You might want to put the resources/file sizes-updating logic in the same area.

And instead of overwriting `transferredChannel`'s properties, you might want to use the SET_TRANSFERRED_CHANNEL mutation to basically replace it with what is `availableChannels[i]` in your loop here.

https://github.com/learningequality/kolibri/blob/e37e8237ac19bce6e8d214eda5f333057f93a107/kolibri/plugins/device_management/assets/src/state/mutations/contentWizardMutations.js#L25